openreview/conference/builder.py

```diff
@@ -473,6 +474,48 @@ def create_desk_reject_invitations(self, reveal_authors=False, reveal_submission
 
         return self.invitation_builder.set_desk_reject_invitation(self, reveal_authors, reveal_submission)
 
+    def create_paper_groups(self, authors=False, reviewers=False, area_chairs=False):
+
+        notes_iterator = self.get_submissions(sort='number:asc', details='original')
+        author_group_ids = []
+
+        for n in notes_iterator:
+            # Paper group
+            group = self.__create_group(
+                group_id = '{conference_id}/Paper{number}'.format(conference_id = self.id, number = n.number),
+                group_owner_id = self.get_area_chairs_id(number = n.number) if self.use_area_chairs else self.id,
```

Comment:
Just removing some spaces for convention.
```suggestion
                group_id = '{conference_id}/Paper{number}'.format(conference_id=self.id, number=n.number),
                group_owner_id = self.get_area_chairs_id(number=n.number) if self.use_area_chairs else self.id,
```